.github/workflows/integration_tests_deployment.yml

```diff
@@ -0,0 +1,68 @@
+name: Integration Tests Deployment
+on:
```

Comment:
Do you want this to run on pull requests too? Perhaps upon approval?

Reply:
This has to run after a Terraform deployment, as that's what it's testing. We may in future want to trigger the dev.yml workflow from a PR - however I'm somewhat cautious about allowing execution within one of my accounts unless I can review or have a level of trust in the individual.
Let's leave it for now and potentially revisit later.

Reply:
Have you considered using the owasp aws account instead?